bucket/lockhunter.json

```diff
@@ -2,19 +2,71 @@
     "version": "3.4.3",
     "description": "Delete files blocked by something you do not know.",
     "homepage": "https://lockhunter.com/index.htm",
-    "license": "Unknown",
+    "license": "Freeware",
```

Comment:
From the website, https://lockhunter.com/download.htm license is https://lockhunter.com/eula.rtf which claims it is a freeware

Reply:
@pratikpc I agree with liking to the license but there are two things to consider here, the actual official license URL downloads the file which might not be very convenient for many users, the other thing is I remember reading somewhere (maybe on the main repo wiki) that only a few strings are valid for this "license" field, I might be wrong.

Also separate PR makes sense

Reply:
@phanirithvij2000 you can have an object with identifier and url in License as well.

Personally I would support using this approach here because the user should have an ability to read the license even if it downloads a file.

Reply:
The setup does come with that eula.rtf file downloaded so maybe having `"license": "$dir\\eula.rtf"` or if that isn't valid adding a note "The license is located at $dir\\eula.rtf" should suffice I think.